app/controllers/users_controller.rb

```diff
@@ -91,4 +93,12 @@ def require_admin
         redirect_to root_path
       end
     end
+
+    def admin_require
+      if logged_in? and current_user.admin?
```

Comment:
if not == unless

Please update your code block

Reply:
ok